index.html

```diff
@@ -0,0 +1,190 @@
+                 <div class="header-columns"><a href="#" class="header-text">Logout</a></div>
+             </header>
+        </div>
+    </div>
+
+       <!-- navbar section start from here -->
+     <div class="contianer-fluid nav-bar-fluid-container">
+        <div class="container">
+             <nav class="row nav-bar-container">
+                <div class="col-2"> <img class="nav-logo" src="images/logo.svg" alt="Conference logo"></div>
+                <div class="col-10 d-flex justify-content-end"> 
+                    <ul class="nav-list">
+                        <li><a href="./about.html"> About</a></li>
+                        <li><a id="coursesiconindex" href="#main-courses-section"> Courses</a></li>
+                        <li><a id="joiniconindex" href="#"> Join</a></li>
+                        <li><a id="sponsericonindex" href="#"> Sponser</a></li>
+                        <li><a id="latesticonindex" href="#"> Latest</a></li>
+                        <li><a href="#"><button class="nav-bar-btn" type="submit">Register Now!</button></a></li>
+                    </ul>
+                </div>
+        
+             
+             </nav>
+        </div>
+     </div>
+
+     <!-- Main home page section start from here -->
+    <div class="contianer-fluid home-page-fluid-container" >
+        <div class="container-fluid menu-bar-fluid">
+            <div class="row">
+                <div id="wrap" class="wrap">
+                    <div id="bar1" class="menu"></div>
+                    <div id="bar2" class="menu"></div>
+                    <div id="bar3" class="menu"></div>
+                </div>
+            </div>
+            <div id="navlistdisplay1" class="row">
+                <ul class="nav-list-mobile">
+                    <li><a href="./about.html"> About</a></li>
+                    <li><a id="coursesiconmobile" href="#main-courses-section">Main Courses</a></li>
+                    <li><a id="joiniconmobile" href="#"> Join</a></li>
+                    <li><a id="sponsericonmobile" href="#"> Sponser</a></li>
+                    <li><a id="latesticonmobile" href="#"> Latest</a></li>
+                </ul>
+            </div>
+        </div>     
+
+        <div class="container main-home-container">           
+            <div class="row about-heading-2">
+                <h2>AFRICA CULTURAL DANCE CONFERENCE<br>2023</h2>
+            </div> 
+                       
+        </div>
+    </div>
+    <div class="row para-heading" >
+        <p>Come enjoy a joyful harmonious infusion of cultural histories and stories fro more than 50 nations and more than 100 tribal commuities and languages.</p>
+        <h1>2023.10.15 (THU) ~ 16(FRI)</h1>
+    </div>  
+    <!-- Main Courses section start from here -->
+    <div id="main-courses-section" class="contianer-fluid main-program">
+        <div class="container main-program-container ">
+            <div class="row">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please `style` your project using the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) provided by `Microverse`. Your project must be the same as [Behance's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), you must make your project `color`, `shape`, `layout`, and `fonts` as shown on [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Please correct them, please remove necessary spaces down and implement correct hover effect🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/235015656-f3127be7-93b0-4d33-8379-2b76c5f42496.png)